recipes/pygridgen/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+    name: pygridgen
+    version: 0.1.1
+
+source:
+    git_url: https://github.com/phobson/pygridgen.git
+    git_tag: 0.1.1
+
+build:
+    number: 0
+    skip: True  # [win]
+    script: python setup.py install
+
+requirements:
+    build:
+        - python
+        - numpy
+    run:
+        - python
+        - pandas
+        - numpy
+        - matplotlib
+        - pyproj
+        - gridgen
+
+test:
+    imports:
+        - pygridgen
+
+about:
+    home: https://github.com/hetland/pygridgen
+    license:  MIT
```

Comment:
Can we leave a comment here referencing this issue ( https://github.com/hetland/pygridgen/issues/27 ) so that we remember to check and make sure this is ok?